devsiteHelper.py

```diff
@@ -231,9 +231,10 @@ def renderDevSiteContent(content, lang='en'):
     fbContent = re.search(r'({%[ ]?framebox.+?%})(.*?){%[ ]?endframebox[ ]?%}(?ms)', framebox)
     fbOpenTag = fbContent.group(1)
     fbHeight = re.search(r'height="(.*?)"', fbContent.group(1))
+    fbClass = re.search(r'class="(.*?)"', fbContent.group(1))
     fbContent = fbContent.group(2)
     fbMemcacheKey = '/framebox/' + hashlib.md5(fbContent).hexdigest()
-    replaceWith = '<iframe class="framebox inherit-locale" '
+    replaceWith = '<iframe class="framebox inherit-locale ' + fbClass.group(1) + '" '
```

Comment:
If no `class` is found, `fbClass` ends up being `None`, and you can't call `fbClass.group(1)` on a `None`.